CODEOWNERS

```
@@ -276,6 +276,7 @@ build.json @home-assistant/supervisor
 /tests/components/command_line/ @gjohansson-ST
 /homeassistant/components/compensation/ @Petro31
 /tests/components/compensation/ @Petro31
+/homeassistant/components/compit/ @CompitHomeAssistant
```

```suggestion
/homeassistant/components/compit/ @Przemko92
```
this should be the username that will be the integration owner